app/utils/database.py

```diff
@@ -366,6 +366,17 @@ class Breeding(BaseModel):
     birth_notes = TextField(null=True)
     litter_size = IntegerField(null=True)
 
+    def as_dict(self):
+        """
+        Returns the objects key/value pair as a dictionary, using parent numbers
+        instead of database id's.
+        """
+        data = super().as_dict()
+        #pylint: disable=no-member
+        data['mother'] = self.mother.number
+        data['father'] = self.father.number
+        return data
+
```

Comment:
Can it be clarified why this is needed?

Reply:
Sure! it's to get the number instead of database id from the foreign keys, but I'll add that to the docstring.

Reply:
Sorry, but doesn't the as_dict from the super class do this already? (I haven't checked, but would expect so.)